### PR TITLE
add didispeech-2

### PR DESCRIPTION
**Pull request by yulan922**

### CODE REVIEW

File: examples/asr/didispeech-2/local/prepare_data.py [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+        logging.info(
+            "Successfully downloaded %s, size(bytes): %d" % (url, statinfo.st_size)  # size -->bytes
+        )
+        with tarfile.open(tar_filepath, "r") as tar:
+            tar.extractall(directory)
+    finally:
+        GFILE.Remove(tar_filepath)
+        
+def is_alphabet(uchar):
+    """判断一个unicode是否是英文字母"""
```
Comment (Some-random) [inline examples/asr/didispeech-2/local/prepare_data.py:66]: use english for comments!

File: examples/asr/didispeech-2/local/prepare_data.py [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+    new_char = uchar.lower()#转小写
+    inside_code = ord(new_char)
+    if inside_code == 0x3000:
+        inside_code = 0x0020
+    else:
+        inside_code -= 0xfee0
+    if inside_code < 0x0020 or inside_code > 0x7e: #转完之后不是半角字符返回原来的字符
+        return new_char
+    return chr(inside_code)
+def stringpartQ2B(ustring):
```
Comment (Some-random) [inline examples/asr/didispeech-2/local/prepare_data.py:88]: newline

File: examples/asr/didispeech-2/local/prepare_data.py [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+        download_and_extract(dircetory, URL)
+        
+    convert_audio_and_split_transcript(dircetory)
+    split_train_dev_test(dircetory)
+    logging.info("Finished processing didispeech csv ")
+    
+if __name__ == "__main__":
+    logging.set_verbosity(logging.INFO)
+    INPUT_DIR = sys.argv[1] 
+    processor(INPUT_DIR)
```
Comment (Some-random) [inline examples/asr/didispeech-2/local/prepare_data.py:179]: after we released download link for didispeech, please add download scripts here

**Review by tjadamlee — APPROVED** [review]
LGTM

**Pull request merged**